Package.swift

```diff
@@ -24,6 +24,28 @@ let package = Package(
         "absl/strings/str_format_test.cc",
         "absl/strings/ascii_test.cc",
         "absl/strings/escaping_test.cc",
+        "absl/strings/match_test.cc",
```

Comment:
Would it be better to not mirror the test.cc and benchmark.cc files into the repo in the first place, to make the Package.swift simpler and avoid having to keep adding new ones on an ongoing basis?

Reply:
nice, I was able to batch remove most of the testing & benchmark files. Updated